File: Web programiranje/Web blanketi/Jun2-2022 - reseno komplet/backend/Models/Vozilo.cs
namespace WebTemplate.Models;

public class Vozilo
{
    [Key]
    public int ID { get; set; }
    [MaxLength(50)]
    public required string Naziv { get; set; }
    public required string SlikaPath { get; set; }
    public int Zapremina { get; set; }
    public int Tezina { get; set; }
    public int Cena { get; set; }
     public DateTime DatumPrijema { get; set; }
     public DateTime DatumIsporuke { get; set; }
     public Kompanija? KompanijinoVozilo { get; set; }
     public List<Roba>? RobaUVozilu { get; set;}
}

File: Web programiranje/Web blanketi/Jun2-2022 - reseno komplet/frontend/app/Vozilo.js
export class Vozilo{
    constructor(id, naziv, slikaPath, zapremina, tezina, cena, datumPrijema, datumIsporuke, kompanijinoVozilo){
        this.id=id;
        this.naziv=naziv;
        this.slikaPath=slikaPath;
        this.zapremina=zapremina;
        this.tezina=tezina;
        this.cena=cena;
        this.datumPrijema=datumPrijema;
        this.datumIsporuke=datumIsporuke;
        this.kompanijinoVozilo=kompanijinoVozilo;
    }
}